apps/judicial-system/web/src/components/RulingInput/RulingInput.tsx

```diff
@@ -14,23 +14,16 @@ import {
 interface Props {
   workingCase: Case
   setWorkingCase: React.Dispatch<React.SetStateAction<Case>>
-  isCaseUpToDate: boolean
   isRequired: boolean
   rows?: number
 }
 
-const RulingInput: React.FC<Props> = (props) => {
-  const {
-    workingCase,
-    setWorkingCase,
-    isCaseUpToDate,
-    isRequired,
-    rows,
-  } = props
-  const { updateCase, autofill } = useCase()
+export const useRulingAutofill = (
```

Comment:
Moving the autofill outside of the component. This way the logic stays in the same place as the input.